Intersect.Client/MonoGame/File Management/MonoContentManager.cs

```diff
@@ -351,14 +351,15 @@ Func<Stream> createStream
                 case ContentTypes.Spell:
                 case ContentTypes.TexturePack:
                 case ContentTypes.TileSet:
-                    return Core.Graphics.Renderer.LoadTexture(name, createStream) as TAsset;
+                    var asset = Core.Graphics.Renderer.LoadTexture(name, createStream) as TAsset;
+                    lookup?.Add(name, asset);
```

Comment:
This might blow up on a duplicate key?

Reply:
Yes. The entire method will also blow up on loading anything but a texture.

Any suggestions on what to do with that?